src/alias.js

```diff
@@ -0,0 +1,45 @@
+const {
+  checkValidAlias,
+  writeJsonToFile,
+} = require("./utils");
+
+function listAliases(configs) {
+  return Object.keys(configs);
+}
+
+async function copyAliasConfig({appConfig, alias, aliasName, configFileName}) {
+  const aliases = listAliases(appConfig.tdxConfigs);
+  if (aliasName === "" || !checkValidAlias(aliasName)) {
```

Comment:
Why isn't aliasName === "" included in the checkValidAlias function?

Reply:
Good point. checkValidAlias already include the regex to check for empty aliasName. So will remove aliasName === ""